week-5/A-dom-manipulation/exercise.js

```diff
@@ -30,12 +30,12 @@ Task 3
 
 When a user clicks the 'Add some text' button, a new paragraph should be added below the buttons that says "Read more below."
 */
-var textButton = document.querySelector("#addTextBtn");
 var paragraph = document.createElement("p");
 paragraph.innerText = "Read more below.";
 paragraph.style.fontSize = "1.5em";
 var addTextArea = document.querySelector(".jumbotron");
-textButton.addEventListener("click", addText);
+document.querySelector("#addTextBtn").addEventListener("click", addText);
```

Comment:
Nice, but remember that you'd have to define paragraph inside the function if you wanted to add more than one

Reply:
I'll wait for the next push before reviewing the practice